packaging/observability/microshift-observability.service

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=MicroShift Observability
+BindsTo=microshift.service
+After=microshift.service
+AssertPathExists=/var/lib/microshift/certs/kube-apiserver-localhost-signer/observability/client.key
+AssertPathExists=/var/lib/microshift/certs/kube-apiserver-localhost-signer/observability/client.crt
+
+[Service]
+ExecStartPre=+/bin/mkdir -p /etc/pki/microshift-observability/
+ExecStartPre=+/bin/cp /var/lib/microshift/certs/kube-apiserver-localhost-signer/observability/client.key /etc/pki/microshift-observability/
+ExecStartPre=+/bin/cp /var/lib/microshift/certs/kube-apiserver-localhost-signer/observability/client.key /etc/pki/microshift-observability/
+ExecStartPre=+/bin/cp /var/lib/microshift/certs/kube-apiserver-localhost-signer/observability/client.crt /etc/pki/microshift-observability/
+ExecStartPre=+/bin/chown -R observability /etc/pki/microshift-observability
+ExecStartPre=+/bin/chmod -R 600 /etc/pki/microshift-observability/
+ExecStartPre=+/bin/chmod 755 /etc/pki/microshift-observability
+
+ExecStart=/usr/bin/opentelemetry-collector --config=/etc/microshift/opentelemetry-collector.yaml
+Restart=always
+User=observability
+TimeoutStartSec=60
+
+# It takes a bit for the certs to be created. This service will reach it's burst limit almost immediately, pretty much
+# guaranteeing that it will reach the restart limit before it can possibly succeed.
+RestartSec=200ms
```

Comment:
We need to optimize this to avoid service failures / restarts. See the unit section comment.